backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/FeatureSupported.java

```diff
@@ -197,6 +197,15 @@ public static boolean isVgpuFramebufferSupported(Version version) {
         return supportedInConfig(ConfigValues.VgpuFramebufferSupported, version);
     }
 
+    /**
+     * Check if vGPU driver parameters are supported
+     *
+     * @param version Compatibility version to check for.
+     */
+    public static boolean isVgpuDriverParametersSupported(Version version) {
+        return Version.v4_7.lessOrEquals(version);
```

Comment:
is there a reason to say that a vm in 4.6 cluster with 4.7 custom compatibility level cannot use it?

Reply:
We need support on the host side to consume the parameter.  If the support is not there, the parameter will be ignored.  We don't prevent using the parameter in older versions but log a warning in such a case.

Reply:
right but that's the thing about custom compatibility level - that it's just for exploring new features before upgrading your cluster, it assumes that the host(s) has been upgraded to support that feature

Reply:
(so it's ok if this fails in case the version of the host that the vm runs on is < 4.5)

Reply:
Do you mean we should drop the warning in logs (and this method)?  I think the warning is useful, I can easily imagine someone setting the parameter while forgetting about the requirement and then wondering why it doesn't work.

Reply:
As clarified offline, the issue here is that if the parameter is used on an incompatible host, nothing will fail with an error, the parameter will be just *silently* ignored.  So it's good to have the warning in engine.log.